### ISSUE TITLE
Same presignature ID reused 80+ times in an hour to generate signature

### ISSUE THREAD

**Issue by ppca** [near/mpc#924]
[log link](https://console.cloud.google.com/logs/query;query=resource.type%3D%22gce_instance%22%0AjsonPayload.presignatureId%3D%229223372036854776000%22%0Aresource.labels.instance_id%3D%226206908420839694874%22%0AjsonPayload.message%3D%22joining%20protocol%20to%20generate%20a%20new%20signature%22;lfeCustomFields=jsonPayload%252Ftarget,jsonPayload%252FpresignatureId;cursorTimestamp=2024-11-06T17:30:11.215084524Z;startTime=2024-11-06T16:31:13.302Z;endTime=2024-11-06T17:31:13.302Z?project=near-cs-dev)
Basically from 8:31 am Nov 6 - 9:31 am Nov 6 PST, log entries for dev node 0 where the presignature_id = `9223372036854776000`, has 89 times `joining protocol to generate a new signature`, and has 46 times `completed signature generation`.

This should not happen because we should not be reusing the presignatures at all.

**Comment by volovyks** [comment]
And it is still there. This pre-signature is still being used to generate a signature.

**Comment by volovyks** [comment]
Something is wrong with our IDs. This ID has both a triple and a presignature, and they are constantly reused.

**Comment by volovyks** [comment]
@ppca please, check the PR, maybe it will shed some lite on this issue
